sdk/nodejs/cmd/pulumi-language-nodejs/language_test.go

```diff
@@ -247,6 +247,22 @@ func TestLanguage(t *testing.T) {
 						t.Skipf("Skipping known failure: %s", expected)
 					}
 
+					// Skip l2-large-string on Node.js 24 https://github.com/nodejs/node/issues/58197
+					// TODO: https://github.com/pulumi/pulumi/issues/19442
+					if tt == "l2-large-string" {
```

Comment:
Skip this only on v24, so we still run it on other versions and don't break things accidentally.